src/main/java/com/example/enterprise/dao/IncomeDAO.java

```diff
@@ -11,11 +11,20 @@
 
     List<Income> allIncomeSources = new ArrayList<>();
 
+    /**
+     * Save an income to the allIncomeSources arraylist
+     * @param income an expense given by user
+     * @throws Exception
+     */
```

Comment:
Added some java doc for the methods with meaningful comments to explain the logic of the method. Java doc is necessary and a good coding convention to keep your code clean. 
To learn more about JavaDoc here is the link: https://www.tutorialspoint.com/java/java_documentation.htm